### ISSUE TITLE
Any route

### ISSUE THREAD

**Issue by Sinevia** [leocavalcante/siler#108]
Can we have a helper method called any:
```php
Route\any('/path', <handler>);
```

**Comment by IvanDelsinne** [comment]
What is its purpose ?

No matter the method, the route would be used ?

**Comment by Sinevia** [comment]
The purpose is if you have POST and GET on single method you dont have to write ['POST','GET']

**Comment by IvanDelsinne** [comment]
Such a method would also catch `PUT`, `PATCH`, `OPTIONS`, `HEAD`.

Would that be intended ?

Anyways, the helper can be developed in [this file](https://github.com/leocavalcante/siler/blob/master/src/Route/Route.php) if you want to make a pull request.

**Comment by Sinevia** [comment]
Yes this is the purpose. It should catch all requests that match the specified path.

**Comment by leocavalcante** [comment]
Added to the roadmap.

Do you folks think we should accept "invalid" verbs? Any verb at all?
Or only those in the HTTP RFC spec?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

**Comment by IvanDelsinne** [comment]
I feel like accepting "anything" (`/\w+/`) would be the best choice.

As an example, I'll take a case I had.

I was working on a small website for home, and I made several functionalities in command line.

To trigger those functionalities, I decided to change the method to `cli` when it was invoked from command line, which allowed me to have routing for both HTTP and CLI, and to share the same back-end code, which finally allowed me to easily switch between command line and HTTP inputs.

**Comment by Sinevia** [comment]
@leocavalcante good shout. Though personally I only use "post" and "get" if its "any" we probably should allow any method, and leave the final implementation to the end user as long as its added to the documentation

**Comment by ndyakov** [comment]
@IvanDelsinne this will have to respond to `DELETED`, `PATCH` , etc... right?

**Comment by IvanDelsinne** [comment]
A "method catchall" route will have to respond to. Or, you can handle the response to throw an "unsupported method" on a given route.
